scheduler-interface-spec.md

```diff
@@ -437,6 +442,8 @@ message AddApplicationRequest {
   // Execution timeout: How long this application can be in a running state
   // 0 or negative value means never expire.
   int64 executionTimeoutMilliSeconds = 6;
+  // The total amount of resources gang placeholders will request
+  Resource placeholderAsk = 7;
```

Comment:
why we need this?
is this a placeholder for future enhancement, e.g headroom check at queue level?

Reply:
It is to make sure that the whole gang fits in the queue, this was already in the design. If the gang does not fit in the queue quota the application will be rejected.
Use case supported: gang total request is 100GB. queue quota is 50 GB. That case we can never allocate the gang, not even in an empty queue and the app will be rejected on submit.

Reply:
Got it, I recall it now.